internal/consensus/pbts_test.go

```diff
@@ -71,34 +71,45 @@ type pbtsTestConfiguration struct {
 	// The setting to use for the TimeoutPropose configuration parameter.
 	timeoutPropose time.Duration
 
-	// The timestamp of the first block produced by the network.
+	// The genesis time
 	genesisTime time.Time
 
-	// The time at which the proposal at height 2 should be delivered.
-	height2ProposalDeliverTime time.Time
+	// The timestamp of the first block produced by the network.
+	height1Time time.Time
```

Comment:
`height1Time` is an intermediate value used during the test, I don't think it really needs to be set as part of configuration of the test in `pbtsTestConfiguration`, so I think we can move it out of this struct and instead into `pbtsTestHarness`.